examples/async-rest/async-rest-webapp/src/main/webapp/WEB-INF/jetty-web.xml

```diff
@@ -9,7 +9,7 @@ org.eclipse.jetty.servlet.WebApplicationContext object
 -->
 
 <Configure class="org.eclipse.jetty.webapp.WebAppContext">
-  <Get class="org.eclipse.jetty.util.log.Log" name="rootLogger">
-    <Call name="warn"><Arg>async-rest webapp is deployed. DO NOT USE IN PRODUCTION!</Arg></Call>
+  <Get name="servletContext">
+    <Call name="log"><Arg>The async-rest webapp is deployed. DO NOT USE IN PRODUCTION!</Arg><Arg></Arg></Call>
```

Comment:
Is this a WARNING level event?
If not, then I don't like this change. :-)

Reply:
It is a warning level event and by passing the null second arg (a Throwable), it remains a warning level event.

Reply:
@gregw I fear that this passes an empty `String` as second parameter.
The way to pass `null` would be `<Arg />`, not `<Arg></Arg>` (which should pass an empty string). Can you double check?